tests/functional/catalogue/test_catalogue.py

```diff
@@ -101,13 +105,14 @@ def test_shows_pagination_navigation_for_multiple_pages(self):
     def test_is_public_on(self):
         product = create_product(upc="grote-bats", is_public=True)
         page = self.app.get(reverse("catalogue:index"))
-        products_on_page = list(page.context["products"].all())
+        products_on_page = list(page.context["products"])
+        products_on_page = [prd.object for prd in products_on_page]
```

Comment:
`prd.object` Is this a breaking change? Is there a way to avoid this

Reply:
Yes, it's a breaking change for the frontend but right now this is just wrong.
the search results are already returned like this so it's now the same across the board.

Reply:
I will add this as an big remark in the next release note